tests/FsCheck.Test/FsCheck.Test.fsproj

```diff
@@ -96,8 +96,14 @@
 	</Target>
 	-->
   <Import Project="..\..\.paket\paket.targets" />
+  <Target Name="BeforeBuild">
+    <ItemGroup>
+      <xunitrunner Include="$(ProjectDir)..\..\packages\xunit.runner.visualstudio\**\*.*" />
+    </ItemGroup>
+    <Copy SourceFiles="@(xunitrunner)" DestinationFolder="$(ProjectDir)..\..\packages\xunit.runner.visualstudio.2.0.0\%(RecursiveDir)" SkipUnchangedFiles="true" />
+  </Target>
```

Comment:
This target should no longer be necessary with the version of paket; it looks like this is a merge problem.